rapids-cmake/cmake/download_with_retry.cmake

```
@@ -0,0 +1,94 @@
+#=============================================================================
+# Copyright (c) 2024-2025, NVIDIA CORPORATION & AFFILIATES.
+# SPDX-License-Identifier: Apache-2.0
+#=============================================================================
+
+#[=======================================================================[.rst:
+rapids_download_with_retry
+--------------------------
+
+.. versionadded:: v25.06.00
+
+Downloads a file from a URL with retry logic for handling network issues.
+
+  .. code-block:: cmake
+
+    rapids_download_with_retry(url output_file sha256 [max_retries] [retry_delay])
```

Match case with the documented parameter names below. Either capitalize these or lowercase the docstring params.
```suggestion
    rapids_download_with_retry(url output_file sha256 [MAX_RETRIES] [RETRY_DELAY])
```

---

Actually, I see now that these are keyword arguments. This should be formatted more like:

```
    rapids_download_with_retry(url output_file sha256 [MAX_RETRIES <max_retries>] [RETRY_DELAY <retry_delay>])
```